src/worker-helpers.js

```diff
@@ -117,9 +117,12 @@ export function findESLintDirectory(modulesDir, config, projectPath, fallbackFor
  */
 export function getESLintFromDirectory(modulesDir, config, projectPath) {
   const { path: ESLintDirectory } = findESLintDirectory(modulesDir, config, projectPath)
+  console.log(ESLintDirectory)
   try {
     // eslint-disable-next-line import/no-dynamic-require
-    return require(ESLintDirectory)
+    const x = require(ESLintDirectory)
+    console.log(x)
```

Comment:
This `require` doesn't work in the tests! 😶

Reply:
@UziTech Does Jasmine change anything regarding `require` cache or something?

Reply:
Jasmine doesn't but atom-jasmine3-test-runner might